src/components/Menu.tsx

```
@@ -0,0 +1,66 @@
+            <div className="z-10 col-span-1 col-start-1 row-start-1 flex h-full w-full items-center justify-start bg-gradient-to-r from-black/80 via-transparent to-transparent p-1">
+              <h2 className="font-display self-justify-end p-4 text-7xl text-white">
+                {section.Name}
+              </h2>
+            </div>
+            <img
+              src={section.ImageUrl}
+              alt={section.Name}
+              className="col-span-1 col-start-1 row-start-1 h-full w-full object-cover object-center"
+            />
+          </div>
+          <div className="my-4 grid grid-cols-[repeat(auto-fit,minmax(350px,1fr))] gap-4">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)